packages/data/lib/src/request/trakt_api/constants/endpoints.dart

```diff
@@ -8,4 +8,6 @@ class TraktApiEndpoints {
   static const String movies = '/movies';
 
   static const String people = '/people';
+
+  static const String comments = '/comments';
```

Comment:
no need to specify a string

Reply:
disagree.
This String is required for request

Reply:
I mean, direct indication of the type of the variable

Reply:
I think, here is optional, yes, we can skip type specify.